examples/reshare/src/application/types.rs

```diff
@@ -132,5 +132,5 @@ where
     C: Signer,
     V: Variant,
 {
-    Block::new(H::EMPTY, 0, None)
+    Block::new(<<H as Hasher>::Digest as Digest>::EMPTY, 0, None)
```

Comment:
`Block` could be re-written to be parameterized on `D: Digest` instead of `H: Hasher` but I opted to make the minimal diff change here